packages/calypso-e2e/src/lib/pages/themes-page.ts

```diff
@@ -59,7 +59,9 @@ export class ThemesPage {
 		await this.pageSettled();
 
 		const searchInput = await this.page.waitForSelector( selectors.searchInput );
-		await Promise.all( [ this.page.waitForNavigation(), searchInput.fill( keyword ) ] );
+		await searchInput.fill( keyword );
+		await searchInput.press( 'Enter' );
```

Comment:
**comment:non-blocking**: Playwright maintainers recommend the `waitForNavigation` calls together with the action that triggers the navigation to ensure that promises accurately resolve. 

It is rare, but in some situations it is possible to have the `waitForNavigation` call be orphaned if the navigation itself completes unusually quick.

Reply:
Thanks for the comment @worldomonation ! I did not know the exact reason why `Promise.all` was used there but now you explain it makes total sense. I am going to change it accordingly